modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js

```diff
@@ -145,15 +145,8 @@ export function Page({
 
 Page.displayName = 'DefaultVariant.Page';
 
-export function PageHeader({description, title}) {
-	return (
-		<>
-			{title && <h2 className="lfr-ddm-form-page-title">{title}</h2>}
-			{description && (
-				<h3 className="lfr-ddm-form-page-description">{description}</h3>
-			)}
-		</>
-	);
+export function PageHeader({title}) {
+	return <>{title && <h2 className="lfr-ddm-form-page-title">{title}</h2>}</>;
```

Comment:
Hello, @larissacribeiro, I've seen that removing the `description` here, it also removes it from the **published Form pages**. Just to make sure, this ticket is just related to Web Content, right? If so, you should make some restriction just to Web Content.
@carolmariaabb, could you confirm if we should keep the page descriptions on **Forms**? 

Thanks!
![image](https://user-images.githubusercontent.com/26491796/228663931-271f91d8-ef72-4885-b0fc-c140bada384e.png)

Reply:
@alinedoleron, yes we should keep the page descriptions on Forms

Reply:
Hi @alinedoleron, I see, gonna work on it now!